chunky/src/java/se/llbit/chunky/renderer/scene/Scene.java

```diff
@@ -2146,8 +2208,9 @@ private synchronized void saveOctree(RenderContext context, TaskTracker taskTrac
 
       boolean saved = false;
       try (DataOutputStream out = new DataOutputStream(new FastBufferedOutputStream(new GZIPOutputStream(context.getSceneFileOutputStream(fileName))))) {
-        OctreeFileFormat.store(out, worldOctree, waterOctree, palette,
-            grassTexture, foliageTexture, waterTexture);
+        //TODO: READD THIS
+//        OctreeFileFormat.store(out, worldOctree, waterOctree, palette,
+//            grassTexture, foliageTexture, waterTexture);
```

Comment:
:bell: Reminder?

Reply:
This will actually be somewhat annoying, as the octree format will have to change
Perhaps an octree internal "2d/3d texture format" would be good, as then we can have implementation defined loading without adjusting the octree format again. Plugins could then register them on load and be asked to load for their registered formats